### PR TITLE
[WEB-5548]: Upgrade PHP coding styles to PSR-12

### PR DESCRIPTION
**Pull request by aosmakserato**
Major changes

The following constants have been given `private` visibility level (excluding changes in tests):

- `src/ClientApp/ClientAppData:: CACHE_KEY`
- `src/ClientApp/ClientAppData:: CACHE_EXPIRY_TIME`
- `src/Slim/Controller/JsonSchemaViewerController:: HTML_VIEW `
- `src/Slim/Controller/JsonSchemaViewerController:: JSON_VIEW `
- `src/Slim/Controller/JsonSchemaViewerController:: JSONLIST_VIEW `
- `src/Slim/Controller/JsonSchemaViewerController:: STYLE_SHEET `

### CODE REVIEW

File: src/Slim/Middleware/AbstractRequestWithAttributeMiddleware.php [hunk 1]
```diff
@@ -21,11 +22,11 @@
...
-    const USER_EMAIL            = 'email';
-    const USER_EMAIL_VERIFIED   = 'email_verified';
-    const REFRESH_TOKEN_ID      = 'rtid';
+    public const APP_ID              = 'app_id';
+    public const APP_NAME            = 'app_name';
+    public const SCOPES              = 'scopes';
+    public const USER_ID             = 'uid';
+    public const USER_EMAIL          = 'email';
+    public const USER_EMAIL_VERIFIED = 'email_verified';
+    public const REFRESH_TOKEN_ID    = 'rtid';
```
Comment (GuillaumeBrousse) [inline src/Slim/Middleware/AbstractRequestWithAttributeMiddleware.php:31]: is this used by other apps? 
or only class that abstract from it ?
just not quite sure about public for `app_id` and `refresh_token_id`
maybe `protected` seems less sensible?

**Review by GuillaumeBrousse — COMMENTED** [review]
LGTM, only one concern

File: src/Slim/Middleware/AbstractRequestWithAttributeMiddleware.php (hunk 1 above)
Comment (aosmakserato) [reply src/Slim/Middleware/AbstractRequestWithAttributeMiddleware.php:31]: `refresh_token_id `
https://github.com/serato/web-sws-php-bootstrap/blob/master/tests/Slim/Middleware/AccessScopes/AccessTokenTest.php#L259

`app_id `
https://github.com/serato/web-sws-php-bootstrap/blob/master/tests/Slim/Middleware/AccessScopes/AccessTokenTest.php#L253

Comment (GuillaumeBrousse) [reply src/Slim/Middleware/AbstractRequestWithAttributeMiddleware.php:31]: all good then

**Review by Jasmin12 — APPROVED** [review]
Nice work @aosmakserato !

**Pull request merged**